build/update_csv.sh

```diff
@@ -1,3 +1,4 @@
-#!/usr/bin/env sh
-CSV_VERSION=${CSV_VERSION:-0.1.1}
-operator-sdk olm-catalog gen-csv --csv-version ${CSV_VERSION} --operator-name service-assurance-operator --update-crds
+#!/usr/bin/env bash
+source "$(dirname "$0")/metadata.sh"
+
+operator-sdk olm-catalog gen-csv --csv-version "${CSV_VERSION}" --operator-name service-assurance-operator --update-crds
```

Comment:
I would prefer a version check in here because it will fail with !=0.12.0

Reply:
A version check on operator-sdk? I can work on that.

@pleimer FYI, when working on the build container; we need specifically operator-sdk 0.12.0